src/index.tsx

```diff
@@ -101,11 +101,23 @@ const EdgeeSdk = ({ src, dataInline }: EdgeeSdkProps): JSX.Element => {
   return script;
 };
 
-// EdgeeContextObject is an interface representing the context object to display in the page.
-export interface EdgeeContextObject {
+// EdgeeDataLayerObject is an interface representing the data layer to be set up
+export interface EdgeeDataLayerObject {
```

Comment:
@SachaMorard eventually it would be cool if we provided a more structured/intuitive way of defining the data layer object (rather than the raw JSON structure). 

So a developer could do something like this (with auto-completion and everything):

```
const edgeeDataLayer: EdgeeDataLayerObject = EdgeeDataLayerFactory.create();

// one event per line
edgeeDataLayer
    .addTrackEvent({...})
    .disableComponent('google_analytics');

// or even some magical
edgeeDataLayer
    .addPageEvent({...})
    .addUserEvent({...})
    .setPageContext({...})
    .setUserContext({...})
    .disableComponent('all')
    .enableComponent('google_analytics')
```